src/OpenTelemetry.AutoInstrumentation/Instrumentation.cs

```diff
@@ -270,9 +270,9 @@ private static void AddLazilyLoadedMetricInstrumentations(LazyInstrumentationLoa
         }
     }
 
-    private static void AddLazilyLoadedTraceInstrumentations(LazyInstrumentationLoader lazyInstrumentationLoader, PluginManager pluginManager, IReadOnlyList<TracerInstrumentation> enabledInstrumentations)
+    private static void AddLazilyLoadedTraceInstrumentations(LazyInstrumentationLoader lazyInstrumentationLoader, PluginManager pluginManager, TracerSettings tracerSettings)
```

Comment:
Why this change?

Reply:
same method & `case TracerInstrumentation.GraphQL:`

Reply:
I have missed that. Thanks.